_docs/developer/how_to_contribute.md

```diff
@@ -28,12 +28,26 @@ Be sure to read the [Suggestions for New Developers](/developer#suggestions-for-
 
 * The PR title should be useful and descriptive (not just the issue#).
 
-* Titles of PR, Issues, and commits should be <= 50 characters.
-  *Usually*.
+    * Titles of PR, Issues, and commits should be <= 50 characters.  *Usually*.
+
+    * Suggested format of the PR Title:
+
+      ```
+      [<TYPE>:<MODULE>] <SUBJECT> (#<PR NUMBER>)
+      ```
+
+      Where `<TYPE>` is one of the following:  
+      `Bugfix`, `Documentation`, `Refactor`, `Dependency`, `Testing`, `Feature`, `UI/UX`, `VPAT`, etc.
```

Comment:
We might want to consider using sorter words/abbreviations, as best practices with Git recommend a commit message of 50 characters.

Reply:
I considered it.  The abbreviations can be awkward and no-intuitive to a new/casual user/sysadmin.  Let's try it without abbreviations and revise the list as we go.   Nothing's automated (yet...  not a priority in near term) so we can play around with the list.